agent/src/agent/modules/logger.py

```diff
@@ -5,8 +5,23 @@
 from logging.handlers import RotatingFileHandler
 
 
-def get_logger(name, level=logging.DEBUG, stdout=False):
+levels = {
+    'CRITICAL': logging.CRITICAL,
+    'FATAL': logging.FATAL,
+    'ERROR': logging.ERROR,
+    'WARN': logging.WARNING,
+    'WARNING': logging.WARNING,
+    'INFO': logging.INFO,
+    'DEBUG': logging.DEBUG,
+    'NOTSET': logging.NOTSET,
+}
+
+
+def get_logger(name, level=None, stdout=False):
     logger = logging.getLogger(name)
+
+    if not level:
+        level = levels.get(os.environ.get('LOGGER_LEVEL'), logging.DEBUG)
```

Comment:
you can use `logging.getLevelName` function instead of defining your own dict 
and plz change the default level to INFO
And the constant name should be `LOG_LEVEL`, I think it's a more common name for things like that

Reply:
I think you don't need to duplicate levels and, for example, declaring a function like this should work fine
`def get_logger(name, level=os.environ.get("LOG_LEVEL", logging.INFO), stdout=False):`